engines/tinsel/tinsel.cpp

```diff
@@ -1021,7 +1027,16 @@ Common::Error TinselEngine::run() {
 
 		initGraphics(width, height, &noirFormat);
 
-		_screenSurface.create(width, 432, noirFormat);
+#if defined(USE_TINYGL)
+		TinyGL::createContext(width, 432, noirFormat, 256, false, false);
```

Comment:
Would it be possible to specify a lower value for the `drawCallMemorySize` parameter than the default one?